core/vm/jump_table_test.go

```diff
@@ -1,24 +1,13 @@
 package vm
 
-import (
-	"math/big"
-	"testing"
-
-	"github.com/ethereum/go-ethereum/params"
-)
+import "testing"
 
 func TestInit(t *testing.T) {
-	params.HomesteadBlock = big.NewInt(1)
-
-	jumpTable.init(big.NewInt(0))
-	if jumpTable[DELEGATECALL].valid {
+	if frontierJumpTable[DELEGATECALL].valid {
 		t.Error("Expected DELEGATECALL not to be present")
 	}
 
-	for _, n := range []int64{1, 2, 100} {
-		jumpTable.init(big.NewInt(n))
-		if !jumpTable[DELEGATECALL].valid {
-			t.Error("Expected DELEGATECALL to be present for block", n)
-		}
+	if !homesteadJumpTable[DELEGATECALL].valid {
+		t.Error("Expected DELEGATECALL not to be present")
```

Comment:
Yes indeed

> On 22 Mar 2016, at 10:57, Felix Lange <notifications@github.com> wrote:
> 
> In core/vm/jump_table_test.go:
> 
> >  		t.Error("Expected DELEGATECALL not to be present")
> >  	}
> >  
> > -	for _, n := range []int64{1, 2, 100} {
> > -		jumpTable.init(big.NewInt(n))
> > -		if !jumpTable[DELEGATECALL].valid {
> > -			t.Error("Expected DELEGATECALL to be present for block", n)
> > -		}
> > +	if !homesteadJumpTable[DELEGATECALL].valid {
> > +		t.Error("Expected DELEGATECALL not to be present")
> I think you mean "... to be present".
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>